geode-core/src/main/java/org/apache/geode/internal/cache/tx/RemoteRemoveAllMessage.java

```diff
@@ -366,6 +356,7 @@ public boolean doLocalRemoveAll(final LocalRegion r, final InternalDistributedMe
           public void run() {
             InternalDistributedMember myId = r.getDistributionManager().getDistributionManagerId();
             for (int i = 0; i < removeAllDataCount; ++i) {
+              // TODO: refactor RemoveAllPRMessage since it is being used for non-pr regions
```

Comment:
Recommend filing a ticket for this now and add the ticket # to the TODO. This might be a good guideline for all TODOs(?).

Reply:
Good idea. I filed GEODE-4597 and will remove these TODO comments